contrib/verifybinaries/README.md

```diff
@@ -1,30 +1,77 @@
 ### Verify Binaries
 
-#### Usage:
+#### Preparation
 
-This script attempts to download the signature file `SHA256SUMS.asc` from https://bitcoin.org.
+As of Bitcoin Core v22.0, releases are signed by a number of public keys on the basis
+of the [guix.sigs repository](https://github.com/bitcoin-core/guix.sigs/). When
+verifying binary downloads, you (the end user) decide which of these public keys you
+trust and then use that trust model to evaluate the signature on a file that contains
+hashes of the release binaries. The downloaded binaries are then hashed and compared to
+the signed checksum file.
 
-It first checks if the signature passes, and then downloads the files specified in the file, and checks if the hashes of these files match those that are specified in the signature file.
+First, you have to figure out which public keys to recognize. Browse the [list of frequent
+builder-keys](https://github.com/bitcoin-core/guix.sigs/tree/main/builder-keys) and
+decide which of these keys you would like to trust. For each key you want to trust, you
+must obtain that key for your local GPG installation.
 
-The script returns 0 if everything passes the checks. It returns 1 if either the signature check or the hash check doesn't pass. If an error occurs the return value is 2.
+You can obtain these keys by
+  - through a browser using a key server (e.g. keyserver.ubuntu.com),
+  - manually using the `gpg --keyserver <url> --recv-keys <key>` command, or
+  - you can run the packaged `verifybinaries.py ... --import-keys` script to 
+    have it automatically retrieve unrecognized keys.
 
+#### Usage
 
+This script attempts to download the checksum file (`SHA256SUMS`) and corresponding
+signature file `SHA256SUMS.asc` from https://bitcoincore.org and https://bitcoin.org.
```

Comment:
Note that bitcoin.org only currently offers `22.0`, and it would seem unlikely that it will offer newer versions any in the near future. It also doesn't actually provide `https://bitcoin.org/bin/bitcoin-core-22.0/SHA256SUMS.asc`, so can't even be used for verification? Could make sense to drop it from this script for now.

Reply:
If we decide to drop it, the `--require-all-hosts` option should probably also be reconsidered. Not saying that it should also be dropped (we could and probably _should_ have more than one host in the future), but at least the help text has to be adapted, which currently explicitly mentions both bitcoin.org and bitcoincore.org.